### PR TITLE
graphql init

### PR DESCRIPTION
**Pull request by ssenchenko**
Init template for GraphQLApi

### CODE REVIEW

**Comment by sriram-mv** [comment]
There needs to be a way this template is accessible via the sam init interface. @mildaniel is working on a PR here: https://github.com/aws/aws-sam-cli/pull/5409

File: nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "hello_world",
+  "version": "1.0.0",
+  "description": "hello world sample for NodeJS",
+  "main": "app.js",
+  "repository": "https://github.com/awslabs/aws-sam-cli/tree/develop/samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs",
+  "author": "SAM CLI",
+  "license": "MIT",
+  "dependencies": {
+    "axios": ">=0.21.1"
```
Comment (mildaniel) [inline nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json:10]: Do we need axios as a dependency?

**Review by mildaniel — APPROVED** [review]

File: nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "scripts": {
+    "test": "mocha tests/unit/"
+  },
+  "devDependencies": {
+    "chai": "^4.3.6",
+    "mocha": "^10.1.0"
```
Comment (mndeveci) [inline nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json:17]: Same comment like @mildaniel 's one, do we need this dev dependencies?

File: nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json (hunk 1 above)
Comment (ssenchenko) [reply nodejs18.x/hello-gql/{{cookiecutter.project_name}}/greeter/package.json:10]: Good call, I copied from the existing hello-world function and didn't pay attention to what dependencies are used. I'll remove all that are not used

**Review by mndeveci — APPROVED** [review]

**Pull request merged**